### PR TITLE
Fill out structures

### PR DESCRIPTION
**Pull request by dneto0**
Fixes: #14

### CODE REVIEW

File: content/types/structures/_index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    {"expr": "an_average_infant.height_m",  "type": "f32"},
+    {"expr": "an_average_infant.mass_kg",  "type": "f32"},
+    {"expr": "average_infant_height",  "type": "f32"},
+    {"expr": "all_zeros_person.age_years",  "type": "u32"},
+    {"expr": "all_zeros_person.height_m",  "type": "f32"},
+    {"expr": "all_zeros_person.mass_kg",  "type": "f32"}
+]}'
+---
+
+A structure type is a named grouping of named data members.
```
Comment (dj2) [inline content/types/structures/_index.md:16]: This reads weird with the double `named`. Maybe just `a named grouping of data members` ?

File: content/types/structures/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Declare structures at module scope, separating their field-specifiers with commas.
+
+To get at a member, follow the structure expression with a dot, then the name of the member.
```
Comment (dj2) [inline content/types/structures/_index.md:20]: Maybe `follow the structure variable name with a dot` as `structure expression` doesn't mean much to most people I don't think?

File: content/types/structures/index.wgsl [hunk 3]
```diff
@@ -0,0 +1,23 @@
+struct Person {
+  age_years: u32,
+  height_m: f32,
+  mass_kg: f32, // The last comma is optional
+}
```
Comment (dj2) [inline content/types/structures/index.wgsl:5]: maybe `}  // no semicolon required` ?

Comment (dj2) [reply content/types/structures/index.wgsl:5]: Is it optional?

File: content/types/structures/_index.md (hunk 1 above)
Comment (dneto0) [reply content/types/structures/_index.md:14]: Will update it.

But this does mirror what the spec says...

> A structure is a named grouping of named member values.

File: content/types/structures/_index.md (hunk 2 above)
Comment (dneto0) [reply content/types/structures/_index.md:18]: yeah, i struggled....

File: content/types/structures/index.wgsl (hunk 3 above)
Comment (dneto0) [reply content/types/structures/index.wgsl:4]: yeah, it's optional. You can sprinkle extra semicolons between global-declarations.

**Review by dj2 — APPROVED** [review]

**Pull request merged**